src/cf-forms/usage.md

```diff
@@ -380,3 +380,19 @@ creating a typical site search form.
     </div>
 </div>
 ```
+
+### Basic Multi-select
```

Comment:
Trivial: Sentence case. Also, we're not hyphenating it anywhere else. Perhaps remove the hyphen here?